src/ast.js

```diff
@@ -2673,12 +2673,27 @@ function parsestrplus (c, n) {
     }
 }
 
+const invalidSyntax = /_[eE]|[eE]_|\._|j_/;
+const invalidDecimalLiteral = /_\.|[+-]_|^0_\D|_j/;
+const validUnderscores = /_(?=[^_])/g;
 function parsenumber (c, s, lineno) {
     var neg;
     var val;
     var tmp;
     var end = s.charAt(s.length - 1);
-
+    
+    if (s.includes("_")) {
```

Comment:
Do we have a string polyfill for `includes()`? (I think we're still aiming to support IE10, aren't we?)

Reply:
We could check this with [`indexOf`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/indexOf#Checking_occurrences) instead, sure.